analysis/src/Commands.ml

```diff
@@ -392,6 +392,14 @@ let test ~path =
                                  Printf.printf "%s\nnewText:\n%s<--here\n%s%s\n"
                                    (Protocol.stringifyRange range)
                                    indent indent newText)))
+          | "c-a" ->
```

Comment:
What's c-a?

Reply:
Unambiguous but getting a bit cryptic

Reply:
It's "codemod - AddMissingCases" 😅 Might be make more sense to have the codemod variant derived from the rest of the string too...